apps/admin/frontend/src/components/export_election_package_modal_button.tsx

```diff
@@ -110,8 +110,7 @@ export function ExportElectionPackageModalButton(): JSX.Element {
           mainContent = (
             <P>
               <UsbImage src="/assets/usb-drive.svg" alt="Insert USB Image" />
-              An election package will be saved to the default location on the
-              mounted USB drive.
+              An election package will be saved to the inserted USB drive.
```

Comment:
<img width="555" alt="Screen Shot 2024-10-08 at 12 20 25 PM plus" src="https://github.com/user-attachments/assets/9999c062-ce76-41e0-b668-7fdaef83bfef">
<img width="550" alt="Screen Shot 2024-10-08 at 12 20 36 PM plus" src="https://github.com/user-attachments/assets/9903bc9e-d145-46d6-87c5-84e680e20c7f">

Reply:
We talked about this today at the office independently and will remove it in another pass.